jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 source "${HOMEgfs}/ush/preamble.sh"
+
+export DATAjob="${DATAROOT}/${RUN}marineanalysis.${PDY:-}${cyc}"
```

Comment:
Just a note here, but `PDY` will not be defined here when we go to ops.  It comes from `setPDY` called in `jjob_header.sh`. 
We will have some work to do here, but its fine for now.